Makefile

```
@@ -20,6 +20,7 @@ BENCHMARKS_BASIC_DIR=$(BUILD_BASE_DIR)/$(BUILD_DIR)/velox/benchmarks/basic/
 BENCHMARKS_DUMP_DIR=dumps
 TREAT_WARNINGS_AS_ERRORS ?= 1
 ENABLE_WALL ?= 1
```

```suggestion
ENABLE_WALL ?= 1
SHELL=/bin/bash
```
otherwise this is run in sh (on my system) and sh doesn't know `source` (but put it at the  top^^)